### PR TITLE
Dom Manipulation with Javascript

### PR DESCRIPTION
**Pull request by sainingo**
# Toggle menu functionality Added
> Vanilla javascript used to add functionality to the toggle menu button
- I've added the functionality that when the hamburger menu is clicked it displays the ul menu
- When the close button is clicked it removes the ul tags
- When one of the ul tags is clicked it redirects to the respective part of the page and it closes the ul menu as well.
menu1

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```
Comment (keroloussamy) [inline .eslintrc.json:25]: - [ ] You did a great work adding the `eslintrc.json` file, but you should also update your `linters.yml` in the `.github/workflows` directory so GitHub can check it also like Html and CSS.

Please update your `linters.yml` file using this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file. 

As you see in the pic there's no checks for `eslintrc`
image

File: styles.css [hunk 2]
```diff
@@ -51,6 +52,47 @@ header nav a {
...
+  right: 30%;
+}
+
+.wrapper {
+  margin-top: 10px;
+}
+
+.wrapper.active {
+  margin-top: 120px;
+}
```
Comment (keroloussamy) [inline styles.css]: - [ ] Your menu works fine, but you have to work on your design to match the Figma design. You have to make the background white and just display the close button and the 3 links.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @sainingo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sainingo** [comment]
Fixed the errors as requ
menu1
ested...

**Review by mike2611 — APPROVED** [review]
Hi @sainingo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**